.github/workflows/deploy-admin.yml

```
@@ -1,15 +1,26 @@
-name: Dev CICD
+name: Dev Admin CI/CD
 
 on:
   push:
     branches: 
     - develop
+    paths:
+      - 'application-admin/**'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)